packages/@aws-cdk/aws-eks/README.md

```diff
@@ -49,7 +49,7 @@ cluster.addResource('mypod', {
 
 ### Capacity
 
-By default, `eks.Cluster` is created with x2 `m5.large` instances.
+By default, `eks.Cluster` is created with a managed nodegroup with x2 `m5.large` instances.
```

Comment:
Sorry for making it a bit hard. But i am concerned about the breaking change. 

Perhaps we can offer some @deprecated opt-in switch `legacyDefaultCapacityXxx` in `ClusterProps` to use the old EC2 capacity instead of Nodegroups?  This way, people will be able to work around the potential breakage.